ecl/hql/hqlattr.cpp

```diff
@@ -3776,6 +3776,46 @@ ITypeInfo * setStreamedAttr(ITypeInfo * _type, bool setValue)
     }
 }
 
+//---------------------------------------------------------------------------------
+IHqlExpression * queryLikelihoodExpr(IHqlExpression * expr)
+{
+    switch(expr->getOperator())
```

Comment:
This function, or that function that calls it needs to cache the result in meta as queryLikelyhood does now.